robyn/argument_parser.pyi

```diff
@@ -0,0 +1,16 @@
+import argparse
+
+
+class ArgumentParser(argparse.ArgumentParser):
+
+    def __init__(self) -> None:
+        pass
```

Comment:
You can use `...` instead of `pass`, and when you add a docstring, you don't need any. JFYK

```suggestion
        ...
```